Tour-Collective-Devs/tc_backend
# Js crew member profile

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes #
No ticket created

## Description of Proposed Changes
Added a url field so that the front end can use it to post with a url.

## Steps to Test
1. Pull down the code
1. Run your server
1. Make sure it works well with the front end
1. 😄 

Outline the steps to test

```
git fetch --all
git checkout js_crew_member_profile
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

## Mentions @username

Tag users that need to review this code

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
1. Every class must be documented with purpose, author, and methods.
1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
